shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -478,7 +478,7 @@ - (instancetype)initWithSemanticsObject:(SemanticsObject*)object {
     if (controller) {
       _platformView = [controller->GetPlatformViewByID(object.node.platformViewId) view];
     }
-    self.accessibilityElements = @[ _semanticsObject, _platformView ];
+    // self.accessibilityElements = @[ _semanticsObject, _platformView ];
```

Comment:
I'll take a second pass at this PR, but if this code turns out to be unnecessary, we'll want to delete this line, as we don't check in commented out code.